### ISSUE TITLE
Rename Keras Metrics

### ISSUE THREAD

**Issue by dgrahn** [keras-team/keras#12480]
I'm calculating a per-class accuracy in my model. It currently outputs, a little bit like this.

    acc: xxx, acc_1: xxx, acc_2: xxx

Is there a way to make this output names like this?

    accuracy/class_0: xxx, accuracy/class_1: xxx, accuracy/class_2: xxx

- [x] Check that you are up-to-date with the master branch of Keras. You can update with:
`pip install git+git://github.com/keras-team/keras.git --upgrade --no-deps`
- [x] Check that your version of TensorFlow is up-to-date. The installation instructions can be found [here](https://www.tensorflow.org/get_started/os_setup).
- [x] Provide a link to a GitHub Gist of a Python script that can reproduce your issue (or just copy the script here if it is short).

**Comment by janithwanni** [comment]
I believe you can write your own keras metric function where you take the array of true values and the predicted values. from the function itself call the keras accuracy metric but print it out with your own specified format. I remember that was possible a few months ago. Have you checked the metrics documentation?

**Comment by dgrahn** [comment]
@janithwanni I have read the docs. That would require a seperate function be written for each Keras metric. I'm trying to avoid duplicating code. Additionally, I'm certain you can't use / in Python function names.
